src/store/Reducers/user.ts

```diff
@@ -0,0 +1,39 @@
+import {createAsyncThunk, createSlice} from '@reduxjs/toolkit';
+
+import {STORAGE_KEYS, storeConstants} from '@constants';
+import {addData} from '@network/apiMethods';
+import {getUserDataUrl} from '@network/apiUrl';
+import {getData, storeData} from '@storage';
+
+const {name, thunk} = storeConstants.user;
+
+export const fetchUserbyEmail = createAsyncThunk(
+  thunk.fetchUserbyEmail,
```

Comment:
`fetchUserbyEmail` naming doesn't look correct "fetch" sound as fetching from API, here you are getting details from storage, update it to something relevant